src/renderer/shared/api/governance/on-chain/service/governanceService.ts

```diff
@@ -39,32 +39,42 @@ async function getReferendums(api: ApiPromise): Promise<Referendum[]> {
 
     if (referendum.isOngoing) {
       const ongoing = referendum.asOngoing;
+      const deciding = ongoing.deciding.unwrapOr(null);
+      const decisionDeposit = ongoing.decisionDeposit.unwrapOr(null);
+      const proposal = ongoing.proposal.isInline
+        ? ongoing.proposal.asInline.toHex()
+        : ongoing.proposal.isLookup
+        ? ongoing.proposal.asLookup.toHex()
+        : ongoing.proposal.isLegacy
+        ? ongoing.proposal.asLegacy.toHex()
+        : '';
```

Comment:
Cascading ternary operator is not good practice, maybe create function with if/return?